acceptance-test/user-pathways/upload-file/upload-file.test.js

```diff
@@ -32,19 +33,43 @@ let page;
 let url;
 
 /**
- * Select the continue button a number of times specified within the
- * supplied parameters
+ * Click on a page element
  *
- * @param {number} loopCount - the number of time the loop should run
+ * @param {string} selector - selector for element on the page
+ *
+ * @returns {Promise}
+ */
```

Comment:
Is it worth putting this function in a module to be reused as you are using it above? Also it has a different name here to the one above